packages/slice-machine/server/src/api/index.ts

```diff
@@ -298,6 +300,24 @@ router.post(
   })
 );
 
+// Sentry Proxy
+// The Sentry client send a POST request with a plaintext body
+// Not supported by express by default
+router.use(function (req, _ /* res */, next) {
```

Comment:
this will happen on every route,

maybe extract the function and use it as middleware on the sentry route

Reply:
@MarcMcIntosh Can I resolve this?

Reply:
yes